src/_core/reducers/models/map.js

```diff
@@ -85,13 +85,13 @@ export const layerModel = Immutable.fromJS({
         tileFunctions: {},
         url: "",
         layer: "",
-        format: "",
-        requestEncoding: "",
-        matrixSet: "",
-        projection: "",
-        extents: [],
+        format: "image/png",
+        requestEncoding: "KVP",
+        matrixSet: undefined,
```

Comment:
What's the reason for `""` -> `undefined`

Reply:
the empty string is valid input for openlayers but doesn't do anything and just results in silent failure. Openlayers should complain about `undefined`

Reply:
Cool.